cmake/crc32_boost.cmake

```diff
@@ -2,4 +2,5 @@ IF(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
     SET(HAVE_CRC32_VPMSUM 1)
     SET(CRC32_VPMSUM_LIBRARY crc32-vpmsum)
     ADD_SUBDIRECTORY(extra/crc32-vpmsum)
+    SET(CRC32_BOOST_LIBRARY ${CRC32_VPMSUM_LIBRARY})
```

Comment:
I believe we should remove `CRC32_VPMSUM_LIBRARY` and use `crc32-vpmsum` directly.

Reply:
Also please use `CRC32_LIBRARY` not to confuse with well known boost.

Reply:
updated